js/views/startScreen.js

```diff
@@ -123,7 +123,7 @@ var StartScreen = React.createClass({
             <span className={playClass} style={playStyle} aria-hidden="true"></span>
           </div>
           <div className="startScreenInfo" style={screenStyle.infoPanel.style}>
-            {titleMetadata}
+            {titleMetadata}<div style={{clear: "both"}}></div>
```

Comment:
Any reason you didn't do it like this?

<div style={{clear: "both"}}>{titleMetadata}</div>